calibrations/mbd/prorun_mbdcal.sh

```
@@ -0,0 +1,277 @@
+    exit 1
+fi
+
+
+
+# Get input files, concatentate into comma-separated string
+inputs=$1
+shift
+for arg in "$@"; do
+  inputs=${inputs},${arg}
+done
+echo $inputs 
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)